src/db/asset.ts

```diff
@@ -13,6 +13,13 @@ export async function assetsForEach(db: Database, callback: AssetInfoCallback, c
   }
 }
 
+export async function assetForId(db: Database, id: string, callback: AssetInfoCallback) {
+  const assets = await getMockAssets();
+  const asset = assets.find((asset) => asset.id === id);
+
+  await callback(asset);
```

Comment:
this should return callback(asset) because if it throws within the assetForId function it'll be an uncaught exception. The latest nodeJS version will crash the server :)

Reply:
Thanks, I think I need to read up on this aspect. I assumed I could catch any errors in he calling code.